examples/kubernetes/deploy-all.sh

```diff
@@ -88,35 +96,35 @@ mkdir -p ${runtime}
 sed "${sedScript}" ${examplesK8sDir}/namespace.yml > ${runtime}/namespace.yml
 apply_global ${runtime}/namespace.yml
 
-echo
-echo "== Generate Certificates"
-mkdir -p ${sslRuntime}
-apps=(dex osprey apiserver)
-for app in "${apps[@]}"; do
-    ${examplesK8sDir}/../generate-certs.sh ${app} ${runtime} ${properties} ssl
-done
+if [[ "${ospreyAthenticationDisabled}" = true ]]; then
```

Comment:
```suggestion
if [[ "${ospreyAuthenticationDisabled}" = true ]]; then
```
careful with usage, and or docs